task8.py

```diff
@@ -0,0 +1,12 @@
+# sort list
+
+numbers = [2, 5, 6, 1, 3, 8, 9, 10]
+
+for i in range(len(numbers)):
+    for j in range(len(numbers) - 1):
+        if numbers[j] > numbers[j+1]:
+            temp = numbers[j]
+            numbers[j] = numbers[j+1]
+            numbers[j+1] = temp        
```

Comment:
Tell me which sorting algorithm you have used here.

Reply:
It is bubble sort.
1. outer loop is for iteration
2. inner loop is for swapping which check if num[j] is greater than num[j+1] and swap elements if condition is true else move to next element
3. After completion of one loop i will increment and repeat process until list is sorted.